boa/src/builtins/map/ordered_map.rs

```diff
@@ -142,6 +142,10 @@ impl<V> OrderedMap<V> {
         }
     }
 
+    pub fn clear(&mut self) {
+        self.map.truncate(0);
```

Comment:
You can use `self.map.clear()`. The implementation of `clear` is internally `truncate(0)`, and both run at O(n) because they need to run the destructors of all the elements. If you're worried about the allocated memory, you can add a `shrink_to_fit`.